scripts/app.js

```diff
@@ -2,17 +2,26 @@
 
 $(document).ready(function() {
 
+  $('#portfolio').hide();
```

Comment:
Specifically targeting one page at a time here might hinder the scalability of your site in the future. Instead, using a tab class to hide all tabs and then only bringing in the tab you want to view might be better.